src/helpers/gcov.ts

```diff
@@ -0,0 +1,23 @@
+import glob from 'fast-glob'
+import { isProgramInstalled, runExternalProgram } from "./util"
+
+export function generateGcovCoverageFiles(projectRoot: string, include: string[] = [], ignore: string[] = [], gcovArgs = '',) {
```

Comment:
The function should have its return type explicitly noted:
- improves API boundary by establishing a clear public contract
- provides self-documentation